ingress/controllers/nginx-third-party/main.go

```diff
@@ -62,14 +62,24 @@ func main() {
 		glog.Fatalf("Please specify --default-backend")
 	}
 
+	glog.Info("Checking if DNS is working")
+	ip, err := checkDNS(*defaultSvc)
```

Comment:
SG, but why do you even need IP of a service? you can directly map svcName -> endpoints no? or is this for something else